build/ci/Makefile

```diff
@@ -144,3 +144,12 @@ ci-e2e-delete-cluster: vault-gke-creds
     	-e "GKE_SERVICE_ACCOUNT_KEY_FILE=$(GO_MOUNT_PATH)/build/ci/$(GKE_CREDS_FILE)" \
     	cloud-on-k8s-ci-e2e \
     	bash -c "make -C operators set-context-gke delete-gke"
+
+# Run docs build
+ci-build-docs:
+	docker run --rm -t \
+        -v $(ROOT_DIR):$(GO_MOUNT_PATH) \
+        docker.elastic.co/docs/build:1 \
+        bash -c "git clone https://github.com/elastic/docs.git && \
+        	/docs/build_docs.pl --doc $(GO_MOUNT_PATH)/docs/index.asciidoc --out $(GO_MOUNT_PATH)/docs/html && \
```

Comment:
Should this not be:
```sh
/docs/build_docs --doc $GO_MOUNT_PATH/docs/index.asciidoc --out $(GO_MOUNT_PATH)/docs/html --chunk 1
```

?

Reply:
We don't need to run `/build_docs` because it's a wrapper on top of `build_docs.pl`. It builds Docker image locally and then run `build_docs.pl` in image. See https://github.com/elastic/docs/blob/master/build_docs#L229. In our case, we are already using pre-built image.

I might add `--chunk 1`, but as I understand from the description https://github.com/elastic/docs#chunking-in-the-right-place it's more an optimization for really big documentation like Elasticsearch, for example. In our case  it probably doesn't make sense right now.

Reply:
Agreed on the direct invocation of `build_docs.pl`

I was suggesting `--chunk 1` because that is we use in the real docs build as well . See:

https://github.com/elastic/docs/blob/6d4e569d8899d4b5865464d081cdc6d5f6697eb0/doc_build_aliases.sh#L73

I am thinking this verification build should be tracking what we are doing in the real docs build, especially if you take into consideration that you were planning to show the results to users. It would otherwise be confusing if the real docs have one format and the docs build here another.

Reply:
Agree with you. Fixed